packages/react-devtools-shared/src/backend/console.js

```diff
@@ -94,36 +108,31 @@ export function patch(): void {
         try {
           // If we are ever called with a string that already has a component stack, e.g. a React error/warning,
           // don't append a second stack.
+          const lastArg = args.length > 0 ? args[args.length - 1] : null;
           const alreadyHasComponentStack =
-            args.length > 0 && FRAME_REGEX.exec(args[args.length - 1]);
+            lastArg !== null &&
+            (PREFIX_REGEX.exec(lastArg) ||
+              ROW_COLUMN_NUMBER_REGEX.exec(lastArg));
```

Comment:
I tend to prefer .test() since exec is stateful any returns a match object that you're not using here.